### PR TITLE
Fix connection leak on page refresh

### PR DESCRIPTION
**Pull request by yongliu-mdsol**
**Issues**:

1. Connection doesn't get disposed on page refresh (manually refresh hangfire page multiple times)
2. Tag count doesn't get returned when hangfire dashboard calls stats (Tags count doesn't get refreshed until a page reload)

### CODE REVIEW

File: src/Hangfire.Tags/Dashboard/TagDashboardMetrics.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using Hangfire.Dashboard;
```
Comment (rdayrit-mdsol) [inline src/Hangfire.Tags/Dashboard/TagDashboardMetrics.cs:1]: maybe separate pr for this fix

**Comment by yongliu-mdsol** [comment]
@erwin-faceit 

Our QA @ktryniszewski-mdsol found two issues as I described in the description.
This PR would fix the connection leak issue.

File: src/Hangfire.Tags/Dashboard/TagDashboardMetrics.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+namespace Hangfire.Tags.Dashboard
+{
+    internal static class TagDashboardMetrics
+    {
+        public readonly static DashboardMetric TagsCount = new DashboardMetric("tags:count", razorPage =>
+            {
+                long count = 0;
+                using (var tagStorage = new TagsStorage(razorPage.Storage))
+                {
+                    count = tagStorage.GetTagsCount();
```
Comment (ryan-dayrit) [inline src/Hangfire.Tags/Dashboard/TagDashboardMetrics.cs:13]: you can get rid of the count var since GetTagsCount() defaults to 0

**Review by ryan-dayrit — APPROVED** [review]
LG
minor comment on count var

File: src/Hangfire.Tags/Dashboard/TagDashboardMetrics.cs (hunk 2 above)
Comment (yongliu-mdsol) [reply src/Hangfire.Tags/Dashboard/TagDashboardMetrics.cs:13]: good catch

**Pull request merged**